src/with-redux/components/Input.js

```diff
@@ -1,68 +1,20 @@
-import React, { Component } from "react";
-import { validators } from "../validators";
+import React, {Component} from "react";
 
 export class Input extends Component {
```

Comment:
in case we don't want to use 'state' in component it's always better to have  stateless functional components